readme.md

```diff
@@ -75,6 +75,37 @@ The codebase also has a number of reference examples.  These are all marked with
 
 `MAINTENANCE_MODE`: Set this flag on a server environment to stop all user requests to the site, such as when you need to make substantial server updates or run a complex database migration.
 
+
+## Deployment
+Prerequisites:
+- AWS CLI
+```
+pip install awscli
+pip install awsebcli
+```
+
+#### Create a new deployment
+Use [eb-create-environment](https://github.com/zagaran/eb-create-environment/)
+```
+pip install eb-create-environment
+eb-create-environment
+```
+#### Initialize local EB config
+Create local `.elasticbeanstalk/config.yml` file:
+```
+eb init
```

Comment:
Given eb-create-environment, I actually find eb-init the wrong tool to use.  Better to give instructions for adding to `~/.aws/profile`.  eb-create-environment creates an `.elasticbeanstalk/config.yml`, which is the only other thing you need from eb-init.